lib/getLocalChanges.js

```diff
@@ -30,26 +28,24 @@ function getLocalChanges (state, options, docsOrIds) {
     return Promise.reject(errors.NOT_AN_OBJECT)
   }
 
-  console.log('🙊  ' + docsOrIds)
-
-  return Promise.all([
-      localDB.info().then(toSeq),
-      remoteDB.info().then(toSeq)
-  ])
-  .then(function(args) {
-    if(args[0] === args[1]) {
-      return Promise.resolve([])
-    } else {
-      return Promise.resolve(args)
-    }
+  return genReplicationId(localDB, remoteDB, {})
+  .then(function (replicationId) {
```

Comment:
In the other modules, we always add an empty line above `.then` or `.catch`, to have separated blocks. Makes it more readable, I guess :) See here for example: https://github.com/hoodiehq/pouchdb-hoodie-api/blob/master/remove-all.js#L19-L46

Could we do that here as well, in both the lib and tests?